projects/storefrontapp-e2e-cypress/cypress/helpers/my-coupons.ts

```diff
@@ -0,0 +1,236 @@
+import { generateMail, randomString } from './user';
+import { login } from './auth-forms';
+import { standardUser } from '../sample-data/shared-users';
+
+// export const configurableProductCode = '1934793';
```

Comment:
remove comments

Reply:
and why `storefrontapp-e2e-cypress/cypress/integration/regression/my-account/my-coupons.e2e-spec.ts` is a empty file?